examples/visual-tests/amp-by-example/LICENSE

```diff
@@ -181,7 +178,7 @@
    APPENDIX: How to apply the Apache License to your work.
```

Comment:
Based on https://www.apache.org/licenses/LICENSE-2.0, I think we should end the license with `END OF TERMS AND CONDITIONS`.